Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     targets: [
         .binaryTarget(
             name: "CertificateSDKPackage",
-            path: "CertificateSDK.xcframework"
+            path: "CertificateSDKPackage.xcframework"
```

Comment:
Instead of renaming the .xcframework file, let's just modify the `Package.swift` file internal target names.  So line 16 becomes `name: "CertificateSDK"` and line 12 becomes `targets: ["CertificateSDK"]`.

It's a simple two line change in one file, and we don't end up with "...Package" on the framework name.

Reply:
Done 👍